core/iwasm/interpreter/wasm.h

```diff
@@ -836,6 +836,9 @@ struct WASMModule {
        AOTModule structure. */
     uint32 module_type;
 
+    /* the package version, should be WASM_CURRENT_VERSION */
```

Comment:
The comment may be not so accurate, e.g. in the future, WASM_CURRENT_VERSION is upgraded to 2, but here the package_version can be 1 or 2. How about somewhat like the file version read from the wasm file?